src/components/decoration/link.tsx

```diff
@@ -4,21 +4,25 @@ import classNames from 'classnames';
 
 import type { LinkProps as NextLinkProps } from 'next/link';
 
-const linkClassNames = 'text-magenta hover:underline active:text-magenta-light';
-
 interface LinkProps extends React.PropsWithChildren<NextLinkProps> {
   className?: string;
+  disableMagenta?: boolean;
```

Comment:
disableMagenta seems to be unused

Reply:
Can this be removed or are you planning to use this prop? Maybe I'm viewing old code and you already removed it

Reply:
I've updated the CustomLink in `byTheNumbers` to use the prop now so that the links aren't inaccessible as shown below
![Screenshot from 2024-09-21 16-34-21](https://github.com/user-attachments/assets/b5c0d951-9c1e-43f2-a1ef-32df743f65cc)